examples/webgl-sea-level.js

```diff
@@ -0,0 +1,88 @@
+import Map from '../src/ol/Map.js';
+import TileLayer from '../src/ol/layer/WebGLTile.js';
+import View from '../src/ol/View.js';
+import XYZ from '../src/ol/source/XYZ.js';
+import {fromLonLat} from '../src/ol/proj.js';
+
+const key = 'get_your_own_D6rA4zTHduk6KOKTXzGB';
+const attributions =
+  '<a href="https://www.maptiler.com/copyright/" target="_blank">&copy; MapTiler</a> ' +
+  '<a href="https://www.openstreetmap.org/copyright" target="_blank">&copy; OpenStreetMap contributors</a>';
+
+const elevation = new TileLayer({
+  opacity: 0.6,
+  source: new XYZ({
+    url:
+      'https://api.maptiler.com/tiles/terrain-rgb/{z}/{x}/{y}.png?key=' + key,
+    maxZoom: 10,
+    tileSize: 512,
+    crossOrigin: 'anonymous',
+  }),
+  style: {
+    variables: {
+      level: 0,
+    },
+    // band math operates on normalized values from 0-1
+    // so we scale by 255 to align with the elevation formula
+    // from https://cloud.maptiler.com/tiles/terrain-rgb/
+    math: [
+      '+',
+      -10000,
+      [
+        '*',
+        0.1 * 255,
+        [
+          '+',
+          ['*', 256 * 256, ['band', 1]],
+          ['+', ['*', 256, ['band', 2]], ['band', 3]],
+        ],
+      ],
+    ],
+    // use the `level` style variable as a stop in the color ramp
+    colors: [
+      ['var', 'level'],
+      [134, 203, 249, 1],
+      ['+', 0.01, ['var', 'level']],
+      [134, 203, 249, 0],
```

Comment:
Using a transparent colour as the output is, at some point in the rendering pipeline, bleeding uninitialized GPU RAM. 
![imagen](https://user-images.githubusercontent.com/1125786/107958730-be1eb280-6fa2-11eb-8646-ae39cc004fe7.png)

Compositing with the (supposedly) underlying basemap *should* happen, but it's not happening under some race conditions (namely, alt-tabbing browser windows). This is 100% reproducible on my dev machine (with an Intel i620 GPU, if that matters).

Reply:
I'm curious if you are still seeing this issue with the latest build of the example.